definitions/ext-sap_system/definition.yml

```diff
@@ -0,0 +1,24 @@
+domain: EXT
+type: SAP_SYSTEM
+goldenTags:
+  - instrumentation.provider
+
+synthesis:
+  rules:
+  # Telemetry from New Relic SAP integration
+  - identifier: SYS_ID
+    name: SYS_ID
+    encodeIdentifierInGUID: true
+    conditions:
+    - attribute: SAP_ETYPE
+      value: SYSTEM
+    tags:
+      instrumentation.provider:
```

Comment:
You can remove this, instrumentation.provider will be always added as a tag if the attribute is present.

Reply:
Ok. Will remove it.